Core/Filesystem/src/core/nand_core.c

```diff
@@ -106,7 +108,9 @@ int _find_lowest_inode(inode_t *inode) {
         }
     }
     if (found_valid_file) {
+#if NAND_DEBUG
```

Comment:
We should probably move the "#if NAND_DEBUG" inside the DBG_PUT. In a separate PR, of course.